build/crds/generate-crd-docs.sh

```diff
@@ -2,23 +2,24 @@
 
 SCRIPT_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)
 GEN_CRD_API_REFERENCE_DOC_VERSION=v0.3.0
-SCRIPT_CHECK_DOCS_DIFF=$(git diff --name-only --diff-filter=M | grep -Ec '/types\.go$')
 SPECIFICATION_FILE=Documentation/CRDs/specification.md
 
 install_generator() {
-    go install github.com/ahmetb/gen-crd-api-reference-docs@${GEN_CRD_API_REFERENCE_DOC_VERSION}
+  go install github.com/ahmetb/gen-crd-api-reference-docs@${GEN_CRD_API_REFERENCE_DOC_VERSION}
 }
 
 run_gen() {
-  if [[  "$SCRIPT_CHECK_DOCS_DIFF" -gt 0 || "$1" = "--force" ]]; then
-    echo "differences found in types.go, rebuilding specification.md"
+  if [[ $SKIP_GEN_CRD_DOCS = true ]]; then
+    echo "SKIP_GEN_CRD_DOCS is set, skipping CRD docs generation"
+  else
+    echo "rebuilding specification.md"
     install_generator
     gen-crd-api-reference-docs \
```

Comment:
`gen-crd-api-reference-docs` is not being found in my path when I run `make crds`. I was able to workaround by passing the `$(GOBIN)` env to this script and then using it here. 
```suggestion
    ${GOBIN}/gen-crd-api-reference-docs \
```